vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -35,15 +34,16 @@
 from indico_vc_zoom.blueprint import blueprint
 from indico_vc_zoom.cli import cli
 from indico_vc_zoom.forms import VCRoomAttachForm, VCRoomForm
-from indico_vc_zoom.notifications import notify_new_host, notify_host_start_url
-from indico_vc_zoom.util import (UserLookupMode, fetch_zoom_meeting, find_enterprise_email, gen_random_password,
-                                 get_schedule_args, get_url_data_args, update_zoom_meeting, ZoomMeetingType)
+from indico_vc_zoom.notifications import notify_host_start_url
+from indico_vc_zoom.util import (UserLookupMode, fetch_zoom_meeting, find_enterprise_email, get_alt_host_emails,
+                                 gen_random_password, get_schedule_args, get_url_data_args, process_alternative_hosts,
+                                 update_zoom_meeting, ZoomMeetingType)
```

Comment:
isort may want to change the order here ;)

Reply:
ok, not only here.. `isort -rc vc_zoom/indico_vc_zoom/` fixes a few other files as well